examples/time_series/goes_xrs_nrt_data.py

```diff
@@ -1,61 +1,65 @@
-=======================================================
+========================================================================
+Creating a TimeSeries from GOES-XRS near real time data with flare times
+========================================================================
 
-This example will demonstrate how to download and load GOES XRS Near Real Time (NRT) data into a `sunpy.timeseries.TimeSeries`.
+This example will demonstrate how to make use of GOES XRS Near Real Time data.
+This includes the goes XRS timeseries data as well as the flare times.
+The real-time datasets can all be found at https://services.swpc.noaa.gov/json/goes/primary/
 """
 
 import matplotlib.pyplot as plt
 import pandas as pd
 
 from astropy import units as u
+from astropy.time import Time
 
 from sunpy import timeseries as ts
 from sunpy.time import parse_time
 
 ###############################################################################
-# We will start by getting reading the GOES-XRS JSON file using `pandas.read_json`.
+# We will start by getting reading the GOES-XRS JSON file using :func:`pandas.read_json`.
 # This allows us to download the file and load it straight into a `pandas.DataFrame`.
-# This file updates every minute and contains only the last 7 days worth of data.
+# This file updates every minute and contains only the last 3 days worth of data.
+# There is also a file with 7 days worth of data.
 
-goes_json_data = pd.read_json("https://services.swpc.noaa.gov/json/goes/primary/xrays-7-day.json")
+goes_data = pd.read_json('https://services.swpc.noaa.gov/json/goes/primary/xrays-3-day.json')
 
 ###############################################################################
-# XRS collects data in two energy channels, "0.05-0.4nm" and "0.1-0.8nm".
-# We separate these "short" and "long" wavelength readings into two arrays.
-
-# This will get us the short wavelength data.
-goes_short = goes_json_data[goes_json_data["energy"] == "0.05-0.4nm"]
-# This will get us the long wavelength data.
-goes_long = goes_json_data[goes_json_data["energy"] == "0.1-0.8nm"]
+# The recorded flux values alternate between the two XRS energy channels:
+# "0.05-0.4nm" and "0.1-0.8nm". We make a `pivot table<https://en.wikipedia.org/wiki/Pivot_table>`__
+# that naturally rearranges the data into two flux columns. We then rename the
+# columns.
+
+goes_data = goes_data.pivot(
+    index='time_tag',
+    columns='energy',
+    values='observed_flux'
+)
+goes_data.rename(columns={'0.05-0.4nm': 'xrsa', '0.1-0.8nm': 'xrsb'}, inplace=True)
 
 ###############################################################################
-# `sunpy.timeseries.TimeSeries` requires a datetime index which we can get
-# directly and transform into `astropy.time.Time`.
+# `sunpy.timeseries.TimeSeries` requires a datetime index, which we can get by
+# parsing the time strings.
 
-time_array = parse_time(goes_short["time_tag"])
+goes_data.index = Time(list(goes_data.index)).datetime
```

Comment:
```suggestion
goes_data.index = parse_time(list(goes_data.index)).datetime
```
After #7983 is merged